articles/azure/azs-how-use-azure-api.md

```diff
@@ -153,33 +195,32 @@ where:
 
 - `resource-path` is the resource you want to query
 
-- `filter-expression` is an optional list of arguments to filter the results of the query
+- `api-version` is the version of the Azure Stack Hub API being called, for example `api-version=2017-12-01`
```

Comment:
You changed other parameters above to use bold formatting instead of code-inline, so should be consistent here. But we probably need to have a wider discussion about this so that we can make sure we use consistent formatting throughout the Knowledge Centre...